### DISCUSSION TITLE
Benefits of P2P?

### DISCUSSION THREAD

**Discussion by lacymorrow** [socketsupply/socket#807] [Q&A] [answered]
I'd like to understand more about P2P, what it is intended to be used for and why it may be beneficial for cross-platform apps. The only scenario I see is when an app is being used by multiple people in the same location without Internet connectivity. What am I missing?

**Comment by heapwolf** [comment, marked as answer]
Socket's P2P allows unlimited users, who may be anywhere in the world, to communicate directly without any data-limits, without any servers or Cloud services, in real-time and asynchronously. This approach can have a substantial cost savings, improved latency in reduced round trips to the data-center as well as privacy and security upsides.
